Xamarin.Forms.Core.UnitTests/MockPlatformServices.cs

```diff
@@ -170,6 +186,11 @@ public void QuitApplication()
 		{
 
 		}
+
+		public IDispatcher GetDispatcher(Guid windowId)
+		{
+			return new MockDispatcher();
```

Comment:
` D:\a\1\s\Xamarin.Forms.Core.UnitTests\MockPlatformServices.cs(192,15): error CS0246: The type or namespace name 'MockDispatcher' could not be found (are you missing a using directive or an assembly reference?) [D:\a\1\s\Xamarin.Forms.Xaml.UnitTests\Xamarin.Forms.Xaml.UnitTests.csproj`

Reply:
(see build failure)

Reply:
Thanks @sandyarmstrong,

I'm building the project, but for me there were no errors, I do not know if it's something with my visual studio.

I will check.

Reply:
I didn't have build problems because my project has the file MockDispatcher.cs included.
And my current branch (multiple-window-uwp) also has the file. You can check [here](https://github.com/edguer/Xamarin.Forms/blob/multiple-window-uwp/Xamarin.Forms.Core.UnitTests/MockDispatcher.cs).

And the inclusion of the file is in the .csproj:
```C#
<Compile Include = "MockDispatcher.cs" />
```

Can you help me?

Thanks,
Edwin Wachs

Reply:
Yeah it looks fine, I don't know why it's not building. Maybe somebody on the Forms team has a clue?

Reply:
yeah Im confused now too... must be something different with CI

Reply:
Can I help in any way?
There are in VSTS log any additional information that can help us?

Reply:
Can you rebase on master ?

Reply:
was missing a reference to the file on XAML.UnitTests csproj .. i think i fixed it

Reply:
Do you want me to do rebase? or not need?

Reply:
No need i have merged master on the branch.

Reply:
Ok! Thanks! 
If you need anything, let me know!

Reply:
Ok! Thanks! 
If you need anything, let me know!

Reply:
Did have another build error on CI or do we still have the same problem?